src/PKSim.Core/Model/PopulationSimulation.cs

```diff
@@ -58,7 +58,8 @@ public override TBuildingBlock BuildingBlock<TBuildingBlock>()
       /// <summary>
       ///    Returns all parameters that could potentially be defined as advanced parameters in a simulation 
       /// </summary>
-      public virtual IEnumerable<IParameter> AllPotentialAdvancedParameters => ParametersOfType(PKSimBuildingBlockType.Simulation | PKSimBuildingBlockType.Compound | PKSimBuildingBlockType.Event | PKSimBuildingBlockType.Formulation);
+      public virtual IEnumerable<IParameter> AllPotentialAdvancedParameters => 
+         ParametersOfType(PKSimBuildingBlockType.Simulation | PKSimBuildingBlockType.Compound | PKSimBuildingBlockType.Event | PKSimBuildingBlockType.Formulation | PKSimBuildingBlockType.Protocol);
```

Comment:
I added the protocol parameters. 
I think this absolutely makes sense. By default none of the parameter are CanBeVariedInPopulation so they don't show up anyways. But if you go to mobi and change the flag to true, they should appear there.

The next step would be to actually always set the dose for example as variable in pop... but this does not have to be done now

Reply:
Unless somebody starts to vary application start times, should be ok

Reply:
Lol yeah. In that case, selber schuld

Reply:
Lol yeah. In that case, selber schuld

Reply:
Lol yeah. In that case, selber schuld